Assets/WizardsCode/Character/Documentation/uml.png.meta

```diff
@@ -0,0 +1,92 @@
+fileFormatVersion: 2
```

Comment:
This is a meta for an empty file. Either file should be removed or it is corrupt.

Reply:
removed